packages/libs/src/sdk/examples/react/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "audius-react-app-example",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@audius/sdk": "latest",
+    "@audius/harmony": "file:./../../../../../harmony",
```

Comment:
will this work? is the expectation that someone makes an example app in the monorepo only?